src/commands/dev.ts

```diff
@@ -23,6 +23,11 @@ export default class Dev extends BaseCommand {
 
   static description = 'Run your stack locally';
 
+  static examples = [
+    'architect dev ./myComponent/architect.yml',
+    'architect dev --port=81 --browser --debug --secret-file=./myComponent/mySecrets.yml ./myComponent/architect.yml',
```

Comment:
Browser is set to true by default. This should be `--no-browser` to show a change in functionality.

Reply:
updated to use --no-browser